src/gateways/createArchiveLoader.ts

```diff
@@ -9,6 +11,30 @@ interface Extras {
   }>;
 }
 
+const cleanArchiveResponse = (archiveContent: any) => {
+  const chapterSlug = 'chapter-';
+  const chapterTitle = 'Chapter';
+  const appendixSlug = 'appendix-';
+  const appendixTitle = 'Appendix ';
+
+  if (archiveContent.tree) {
+    archiveContent.tree.contents.map((item: any) => {
+      if (item.slug.includes(chapterSlug) || item.slug.includes(appendixSlug)) {
+        item.slug = item.slug.replace(chapterSlug, '').replace(appendixSlug, '');
```

Comment:
if you make these regex like `/^chapter-/` and `/^appendix-/` they'll only match at the beginning of the string, so that is safer, and you can run the replace without a conditional, it'll just do nothing if it doesn't find anything